components/mixins/experience-event/events/statusincampaignprogressionchanged.schema.json

```diff
@@ -29,42 +29,42 @@
                 "xdm:campaignID": {
                   "title": "Campaign ID",
                   "type": "string",
-                  "description": "Unique identifier for campaign."
+                  "description": "Unique ID of the Campaign."
                 },
                 "xdm:newStatusID": {
                   "title": "New Status ID",
                   "type": "string",
-                  "description": "Unique identifier for new status."
+                  "description": "ID of the lead's new status in the campaign."
                 },
                 "xdm:newStatusName": {
                   "title": "New Status Name",
                   "type": "string",
-                  "description": "Friendly name for new status."
+                  "description": "Friendly name of the lead's new status in the campaign."
```

Comment:
Friendly name of the lead’s new status in the campaign. => Name of the lead’s new status in the campaign.